Gemfile.lock

```diff
@@ -56,47 +56,6 @@ PATH
       syslog-logger (~> 1.6)
       tty-screen (~> 0.6)
       uuidtools (~> 2.1.5)
-    chef (15.0.233-universal-mingw32)
```

Comment:
Looks like you just ran bundle update on the gemfile.lock. That removes the windows specific gems. Use the rake task to update the deps instead, but make sure not to bump the bundle version line at the bottom of the file.

Reply:
ah damnit. Thanks.